ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyOptimizer.java

```diff
@@ -437,7 +437,7 @@ private void transitionLayoutOnlyViewToNativeView(
 
   private static boolean isLayoutOnlyAndCollapsable(@Nullable ReactStylesDiffMap props) {
     if (props == null) {
-      return true;
+      return false;
```

Comment:
Some children are null so we can't set collapsable = false on them. I had to add a null check in children map here https://github.com/facebook/react-native/pull/9456/commits/1e87a600e6ba71d7bdcac2c02659988d4f3ddafb#diff-f8f8b220fc0d1573e4f8b78c84415075R455